main.go

```diff
@@ -14,6 +17,7 @@ import (
 var (
 	configFile      = flag.String("config-file", "conf.toml", "Path to configuration file")
 	authFile        = flag.String("auth-file", "auth.toml", "Path to authentication file")
+	dataPath        = flag.String("data-path", "./", "Where to save data")
```

Comment:
Pls use `os.PathSeparator` to prevent using Win/Unix specific directory separator characters !

Reply:
Sounds reasonable but I guess I could leave it out completely, as in every place I use it, it is used in a path.Join

Reply:
is now also implemented with the latest commits